tests/acl/test_acl.py

```diff
@@ -593,6 +591,8 @@ def acl_table(duthosts, rand_one_dut_hostname, setup, stage, ip_version, tbinfo)
             loganalyzer.expect_regex = [LOG_EXPECT_ACL_TABLE_REMOVE_RE]
             with loganalyzer:
                 create_or_remove_acl_table(duthost, acl_table_config, setup, "remove", topo)
+                wait_until(300, 20, 0, check_msg_in_syslog,
```

Comment:
300 seconds look too long to me. Do we really need 5 minutes to wait the ACL table to be created?

Reply:
I lowered the wait time to 1 minute and the polling frequency to every 10 secs.  From my testing on single-asic this should be sufficient to catch the instances when the syslog is delayed and appears after the loganalyzer check.